index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<title>Portfolio</title>
+	<link rel="stylesheet" type="text/css" href="Style.css">
+</head>
+<body>
+	<header>
+		<label class="logo">My Logo</label>
+		<ul>
+			<li><a href="#" class="bar">
+			<div class="b1"></div>
+			<div class="b2"></div>
+			<div class="b3"></div>
+		</a></li>
+			</ul>	
```

Comment:
The `hamburger` icon should be wrapped inside a `nav` element.
You should have something like this:
![image](https://user-images.githubusercontent.com/38088818/164516516-a7167cfe-1783-4c22-b11c-eef8367ad05c.png)

Also, please consider using proper indentation to layout your HTML. This makes your code cleaner and more readable.
The final result of your header after changes should look like this:
![image](https://user-images.githubusercontent.com/38088818/164516988-e4b182f9-752d-4a1e-9b82-e2f72eab6d7c.png)

And the entire `body` of your HTML after indentation can look more like this:
![content](https://user-images.githubusercontent.com/38088818/164517700-2b9dde85-d097-4fc4-b112-adc369da782d.png)

Notice how I also changed the `div` element to `main` on line 25? That's because it is more semantic to wrap the main content of the body in `main` rather than a `div`.

Kindly make these final changes and you are good to go. 👍🏾
Cheers and happy coding 🥂